src/LeanMapper/DefaultEntityFactory.php

```diff
@@ -19,7 +19,7 @@
 class DefaultEntityFactory implements IEntityFactory
 {
 
-    /*
+    /**
```

Comment:
Nevím, jak je to dnes, ale kdysi vyžadoval ApiGen zápis jen s jednou hvězdičkou. Lean Mapper stále ještě má API dokumentaci online. Je otázkou, jestli na to rezignovat nebo ne. :)